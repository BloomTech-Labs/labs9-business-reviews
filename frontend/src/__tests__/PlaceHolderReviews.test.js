import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';
import PlaceHolderReviewsComponent from '../components/PlaceHolderReviews';

describe('PlaceHolderReviewsComponent />', () => {
  it('renders', () => {
    shallow(<PlaceHolderReviewsComponent />);
  });
  it('matches snapshot', () => {
    const wrapper = shallow(<PlaceHolderReviewsComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
