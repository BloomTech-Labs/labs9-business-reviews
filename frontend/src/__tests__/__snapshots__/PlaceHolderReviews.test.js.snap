// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaceHolderReviewsComponent /> matches snapshot 1`] = `
<Fragment>
  <div
    className="review"
  >
    <div
      className="review-img1"
    />
    <p>
      <span
        aria-label="stars"
        role="img"
      >
        ⭐⭐⭐
      </span>
    </p>
    <h3>
      @eddbunk
    </h3>
  </div>
  <div
    className="review"
  >
    <div
      className="review-img2"
    />
    <p>
      <span
        aria-label="stars"
        role="img"
      >
        ⭐⭐⭐⭐⭐
      </span>
    </p>
    <h3>
      @alixjones
    </h3>
  </div>
  <div
    className="review"
  >
    <div
      className="review-img3"
    />
    <p>
      <span
        aria-label="stars"
        role="img"
      >
        ⭐⭐
      </span>
    </p>
    <h3>
      @carloG
    </h3>
  </div>
</Fragment>
`;
